### PR TITLE
Formatting fixes and cleanup

### PR DESCRIPTION
**Pull request by FiniteStateGit**
Fixes Issues: https://github.com/o3de/o3de.org/issues/190 https://github.com/o3de/o3de.org/issues/205 https://github.com/o3de/o3de.org/issues/206 https://github.com/o3de/o3de.org/issues/207 https://github.com/o3de/o3de.org/issues/208 https://github.com/o3de/o3de.org/issues/273 https://github.com/o3de/o3de.org/issues/290

Also added note shortcodes, removed several links to non-existent pages, fixed anchor.

### CODE REVIEW

File: content/docs/user-guide/components/reference/physx/rigid-body-physics.md [hunk 1]
```diff
@@ -123,10 +123,11 @@ A PhysX entity that is dynamic can move and collide with other entities.
 1. Create another entity underneath the first, with a **PhysX Collider** component and a **Mesh** component but without a **PhysX Rigid Body** component. This will create a static rigid body.
 
 1. Press **Ctrl+G** to enter gameplay mode.
-**Example**
 
-   The entity has a **PhysX Collider** component and a **PhysX Rigid Body** component attached. Because the object is dynamic, it falls and then collides with the static rigid body.
-![\[Animated example of a dynamic entity falling to the terrain and bouncing around.\]](/images/user-guide/shared/physx-creating-dynamic-object-1.gif)
+    ![\[Animated example of a dynamic entity falling to the terrain and bouncing around.\]](/images/user-guide/shared/physx-creating-dynamic-object-1.gif)
```
Comment (sptramer) [inline content/docs/user-guide/components/reference/physx/rigid-body-physics.md:127]:
```suggestion
    ![Animated example of a dynamic entity falling to the terrain and bouncing around.](/images/user-guide/shared/physx-creating-dynamic-object-1.gif)
```

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md [hunk 2]
```diff
@@ -7,40 +7,48 @@ Creating a vegetation layer is the first and most basic step in creating your dy
 
 **To create a vegetation layer**
 
-1. [Create an entity](/docs/userguide/creating-entity.md) and name it.
+1. Create an entity(/docs/userguide/creating-entity.md) and name it.
```
Comment (sptramer) [inline content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md:10]:
```suggestion
1. Create an entity and name it.
```

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/selection-random.md [hunk 3]
```diff
@@ -6,67 +6,77 @@ title: Creating Weight-Based Random Selection
...
+        Repeat this step with different assets until you have three or more assets.
+
+        Although you have specified multiple assets, only the first asset that you added appears in the Viewport. The next several procedures show you how to distribute the vegetation so that all of them appear.
 
 1. Click **Add Component**, **Vegetation Asset Weight Selector**.
 
    In the next step, [Creating a Gradient Entity](#create-gradient-entity), you link this component to the gradient, which supplies values between 0.0 to 1.0 at a given position. Assets are then mapped based on those values and each asset's weight and order values.
 
-## Creating a Gradient Entity 
+## Creating a Gradient Entity {#create-gradient-entity}
```
Comment (sptramer) [inline content/docs/user-guide/gems/reference/environment/vegetation/procedures/selection-random.md:43]: Avoid the use of manually-defined anchors in headers; the autogenerated anchor here is named `creating-a-gradient-entity`.

```suggestion
## Creating a Gradient Entity
```

**Review by sptramer — CHANGES_REQUESTED** [review]
Formatting change fixes. Thanks for your contribution!

**Review by micronAMZN — APPROVED** [review]

**Review by sptramer — APPROVED** [review]

**Pull request merged**